presto-main/src/main/java/com/facebook/presto/cost/ScalarStatsCalculator.java

```diff
@@ -96,5 +105,37 @@ protected ColumnStatistics visitLiteral(Literal node, Void context)
             builder.setNullsFraction(Estimate.zeroValue());
             return builder.build();
         }
+
+        protected ColumnStatistics visitCast(Cast node, Void context)
+        {
+            // todo - are there cases that lowValue/highValue does not map to lowValue/highValue after cast?
```

Comment:
Yeah. We do not care about it right now. As we opeate in double domain and do not support strings.